src/main/kotlin/no/nav/familie/tilbake/vilkårsvurdering/VilkårsvurderingService.kt

```diff
@@ -157,6 +159,19 @@ class VilkårsvurderingService(
         }
     }
 
+    @Transactional
+    fun sjekkOmVilkårsvurderingPerioderErLike(
+        behandlingId: UUID,
+    ): Boolean {
+        val vikårsvurdering = vilkårsvurderingRepository.findByBehandlingIdAndAktivIsTrue(behandlingId)
+        val førstePeriode = vikårsvurdering?.perioder?.first()
+        return if (førstePeriode != null) {
+            vikårsvurdering?.perioder?.all { it.aktsomhet == førstePeriode.aktsomhet && it.godTro == førstePeriode.godTro && it.vilkårsvurderingsresultat == førstePeriode.vilkårsvurderingsresultat && it.begrunnelse == førstePeriode.begrunnelse } ?: false
```

Comment:
Aktsomhet og Godtro har "id" og "sporbar" som vil være forskjellig for alle perioder. 
Det samme gjelder `vilkårsvurderingSærligeGrunner` inni aktsomhet

Reply:
Det er sant, ser det gikk litt fort i svingene her 😅  Vi får lager vi egne metoder pr type.